// strman/lib/strman.js
'use strict';

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.trim = exports.substr = exports.reverse = exports.indexOf = exports.endsWith = exports.binEncode = exports.transliterate = exports.replace = exports.leftTrim = exports.htmlEncode = exports.decEncode = exports.binDecode = exports.toUpperCase = exports.startsWith = exports.repeat = exports.leftPad = exports.htmlDecode = exports.decDecode = exports.between = exports.toStudlyCaps = exports.split = exports.removeSpaces = exports.lastIndexOf = exports.hexEncode = exports.countSubstr = exports.base64encode = exports.toSnakeCase = exports.slugify = exports.removeRight = exports.last = exports.hexDecode = exports.containsAny = exports.base64decode = exports.toLowerCase = exports.slice = exports.removeNonWords = exports.isUpperCase = exports.format = exports.containsAll = exports.at = exports.toKebabCase = exports.shuffle = exports.removeLeft = exports.isString = exports.first = exports.contains = exports.appendArray = exports.urlEncode = exports.toDecamelize = exports.safeTruncate = exports.removeEmptyStrings = exports.isLowerCase = exports.equal = exports.compare = exports.append = exports.urlDecode = exports.toCamelCase = exports.rightTrim = exports.prependArray = exports.insert = exports.ensureRight = exports.collapseWhitespace = exports.truncate = exports.surround = exports.rightPad = exports.prepend = exports.inequal = exports.ensureLeft = exports.chars = undefined;

var _chars2 = require('./chars');

var _chars3 = _interopRequireDefault(_chars2);

var _ensureleft = require('./ensureleft');

var _ensureleft2 = _interopRequireDefault(_ensureleft);

var _inequal2 = require('./inequal');

var _inequal3 = _interopRequireDefault(_inequal2);

var _prepend2 = require('./prepend');

var _prepend3 = _interopRequireDefault(_prepend2);

var _rightpad = require('./rightpad');

var _rightpad2 = _interopRequireDefault(_rightpad);

var _surround2 = require('./surround');

var _surround3 = _interopRequireDefault(_surround2);

var _truncate2 = require('./truncate');

var _truncate3 = _interopRequireDefault(_truncate2);

var _collapsewhitespace = require('./collapsewhitespace');

var _collapsewhitespace2 = _interopRequireDefault(_collapsewhitespace);

var _ensureright = require('./ensureright');

var _ensureright2 = _interopRequireDefault(_ensureright);

var _insert2 = require('./insert');

var _insert3 = _interopRequireDefault(_insert2);

var _prependarray = require('./prependarray');

var _prependarray2 = _interopRequireDefault(_prependarray);

var _righttrim = require('./righttrim');

var _righttrim2 = _interopRequireDefault(_righttrim);

var _tocamelcase = require('./tocamelcase');

var _tocamelcase2 = _interopRequireDefault(_tocamelcase);

var _urlDecode2 = require('./urlDecode');

var _urlDecode3 = _interopRequireDefault(_urlDecode2);

var _append2 = require('./append');

var _append3 = _interopRequireDefault(_append2);

var _compare2 = require('./compare');

var _compare3 = _interopRequireDefault(_compare2);

var _equal2 = require('./equal');

var _equal3 = _interopRequireDefault(_equal2);

var _islowercase = require('./islowercase');

var _islowercase2 = _interopRequireDefault(_islowercase);

var _removeemptystrings = require('./removeemptystrings');

var _removeemptystrings2 = _interopRequireDefault(_removeemptystrings);

var _safetruncate = require('./safetruncate');

var _safetruncate2 = _interopRequireDefault(_safetruncate);

var _todecamelize = require('./todecamelize');

var _todecamelize2 = _interopRequireDefault(_todecamelize);

var _urlencode = require('./urlencode');

var _urlencode2 = _interopRequireDefault(_urlencode);

var _appendarray = require('./appendarray');

var _appendarray2 = _interopRequireDefault(_appendarray);

var _contains2 = require('./contains');

var _contains3 = _interopRequireDefault(_contains2);

var _first2 = require('./first');

var _first3 = _interopRequireDefault(_first2);

var _isstring = require('./isstring');

var _isstring2 = _interopRequireDefault(_isstring);

var _removeleft = require('./removeleft');

var _removeleft2 = _interopRequireDefault(_removeleft);

var _shuffle2 = require('./shuffle');

var _shuffle3 = _interopRequireDefault(_shuffle2);

var _tokebabcase = require('./tokebabcase');

var _tokebabcase2 = _interopRequireDefault(_tokebabcase);

var _at2 = require('./at');

var _at3 = _interopRequireDefault(_at2);

var _containsall = require('./containsall');

var _containsall2 = _interopRequireDefault(_containsall);

var _format2 = require('./format');

var _format3 = _interopRequireDefault(_format2);

var _isuppercase = require('./isuppercase');

var _isuppercase2 = _interopRequireDefault(_isuppercase);

var _removenonwords = require('./removenonwords');

var _removenonwords2 = _interopRequireDefault(_removenonwords);

var _slice2 = require('./slice');

var _slice3 = _interopRequireDefault(_slice2);

var _tolowercase = require('./tolowercase');

var _tolowercase2 = _interopRequireDefault(_tolowercase);

var _base64decode2 = require('./base64decode');

var _base64decode3 = _interopRequireDefault(_base64decode2);

var _containsany = require('./containsany');

var _containsany2 = _interopRequireDefault(_containsany);

var _hexdecode = require('./hexdecode');

var _hexdecode2 = _interopRequireDefault(_hexdecode);

var _last2 = require('./last');

var _last3 = _interopRequireDefault(_last2);

var _removeright = require('./removeright');

var _removeright2 = _interopRequireDefault(_removeright);

var _slugify2 = require('./slugify');

var _slugify3 = _interopRequireDefault(_slugify2);

var _tosnakecase = require('./tosnakecase');

var _tosnakecase2 = _interopRequireDefault(_tosnakecase);

var _base64encode2 = require('./base64encode');

var _base64encode3 = _interopRequireDefault(_base64encode2);

var _countsubstr = require('./countsubstr');

var _countsubstr2 = _interopRequireDefault(_countsubstr);

var _hexencode = require('./hexencode');

var _hexencode2 = _interopRequireDefault(_hexencode);

var _lastindexof = require('./lastindexof');

var _lastindexof2 = _interopRequireDefault(_lastindexof);

var _removespaces = require('./removespaces');

var _removespaces2 = _interopRequireDefault(_removespaces);

var _split2 = require('./split');

var _split3 = _interopRequireDefault(_split2);

var _tostudlycaps = require('./tostudlycaps');

var _tostudlycaps2 = _interopRequireDefault(_tostudlycaps);

var _between2 = require('./between');

var _between3 = _interopRequireDefault(_between2);

var _decdecode = require('./decdecode');

var _decdecode2 = _interopRequireDefault(_decdecode);

var _htmldecode = require('./htmldecode');

var _htmldecode2 = _interopRequireDefault(_htmldecode);

var _leftpad = require('./leftpad');

var _leftpad2 = _interopRequireDefault(_leftpad);

var _repeat2 = require('./repeat');

var _repeat3 = _interopRequireDefault(_repeat2);

var _startswith = require('./startswith');

var _startswith2 = _interopRequireDefault(_startswith);

var _touppercase = require('./touppercase');

var _touppercase2 = _interopRequireDefault(_touppercase);

var _bindecode = require('./bindecode');

var _bindecode2 = _interopRequireDefault(_bindecode);

var _decencode = require('./decencode');

var _decencode2 = _interopRequireDefault(_decencode);

var _htmlencode = require('./htmlencode');

var _htmlencode2 = _interopRequireDefault(_htmlencode);

var _lefttrim = require('./lefttrim');

var _lefttrim2 = _interopRequireDefault(_lefttrim);

var _replace2 = require('./replace');

var _replace3 = _interopRequireDefault(_replace2);

var _transliterate2 = require('./transliterate');

var _transliterate3 = _interopRequireDefault(_transliterate2);

var _binencode = require('./binencode');

var _binencode2 = _interopRequireDefault(_binencode);

var _endswith = require('./endswith');

var _endswith2 = _interopRequireDefault(_endswith);

var _indexof = require('./indexof');

var _indexof2 = _interopRequireDefault(_indexof);

var _reverse2 = require('./reverse');

var _reverse3 = _interopRequireDefault(_reverse2);

var _substr2 = require('./substr');

var _substr3 = _interopRequireDefault(_substr2);

var _trim2 = require('./trim');

var _trim3 = _interopRequireDefault(_trim2);

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }

exports.chars = _chars3.default;
exports.ensureLeft = _ensureleft2.default;
exports.inequal = _inequal3.default;
exports.prepend = _prepend3.default;
exports.rightPad = _rightpad2.default;
exports.surround = _surround3.default;
exports.truncate = _truncate3.default;
exports.collapseWhitespace = _collapsewhitespace2.default;
exports.ensureRight = _ensureright2.default;
exports.insert = _insert3.default;
exports.prependArray = _prependarray2.default;
exports.rightTrim = _righttrim2.default;
exports.toCamelCase = _tocamelcase2.default;
exports.urlDecode = _urlDecode3.default;
exports.append = _append3.default;
exports.compare = _compare3.default;
exports.equal = _equal3.default;
exports.isLowerCase = _islowercase2.default;
exports.removeEmptyStrings = _removeemptystrings2.default;
exports.safeTruncate = _safetruncate2.default;
exports.toDecamelize = _todecamelize2.default;
exports.urlEncode = _urlencode2.default;
exports.appendArray = _appendarray2.default;
exports.contains = _contains3.default;
exports.first = _first3.default;
exports.isString = _isstring2.default;
exports.removeLeft = _removeleft2.default;
exports.shuffle = _shuffle3.default;
exports.toKebabCase = _tokebabcase2.default;
exports.at = _at3.default;
exports.containsAll = _containsall2.default;
exports.format = _format3.default;
exports.isUpperCase = _isuppercase2.default;
exports.removeNonWords = _removenonwords2.default;
exports.slice = _slice3.default;
exports.toLowerCase = _tolowercase2.default;
exports.base64decode = _base64decode3.default;
exports.containsAny = _containsany2.default;
exports.hexDecode = _hexdecode2.default;
exports.last = _last3.default;
exports.removeRight = _removeright2.default;
exports.slugify = _slugify3.default;
exports.toSnakeCase = _tosnakecase2.default;
exports.base64encode = _base64encode3.default;
exports.countSubstr = _countsubstr2.default;
exports.hexEncode = _hexencode2.default;
exports.lastIndexOf = _lastindexof2.default;
exports.removeSpaces = _removespaces2.default;
exports.split = _split3.default;
exports.toStudlyCaps = _tostudlycaps2.default;
exports.between = _between3.default;
exports.decDecode = _decdecode2.default;
exports.htmlDecode = _htmldecode2.default;
exports.leftPad = _leftpad2.default;
exports.repeat = _repeat3.default;
exports.startsWith = _startswith2.default;
exports.toUpperCase = _touppercase2.default;
exports.binDecode = _bindecode2.default;
exports.decEncode = _decencode2.default;
exports.htmlEncode = _htmlencode2.default;
exports.leftTrim = _lefttrim2.default;
exports.replace = _replace3.default;
exports.transliterate = _transliterate3.default;
exports.binEncode = _binencode2.default;
exports.endsWith = _endswith2.default;
exports.indexOf = _indexof2.default;
exports.reverse = _reverse3.default;
exports.substr = _substr3.default;
exports.trim = _trim3.default;
exports.default = exports;
